generators/generatorHelper.js

```diff
@@ -0,0 +1,67 @@
+const timeBoundSuffixes = ['', '_1d', '_7d', '_30d', '_60d', '_90d', '_180d', '_365d', '_2y', '_3y', '_5y', '_10y']
```

Comment:
Remove the underscores and add them in the function generation. Also I prefer the name `timeBounds`, suffixes is implementation detail.

Reply:
Should be okay now